src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
> OCamlPro's website has been 404ing more than once with stuff hosted there

Github has been down more frequently in the last years, and opam-repository is still hosted on it.

> (the last occurence being that #2438)

Indeed, a good example, it was fixed immediately.

> We don't want to wait for an OCamlPro employee to show up to be able to fix these problems.

Funny to read when the one person who fixes bugs in OPAM (Louis Gesbert) is actually an OCamlPro employee.

> I'm sure your sysadmin time will be gladly appreciated on opam.ocaml.org.
￼
It takes me more time to discuss on this forum everything I would like to do than implementing them and starting them on OCamlPro's servers. If your goal is to demotivate outside contributions to OPAM and its repository, you are quite efficient !